app/konnect/api/identity-management/identity-integration.md

```diff
@@ -0,0 +1,170 @@
+---
+title: End to End Identity Integration
+content-type: how-to
+---
+
+
+## Create a custom team
+
+Custom Teams serve as a primary way for organizations to provision access of users to different entities in the organization. Custom teams are used to mirror the organizational structure in an organization, any user who is a member of a custom team will inherit all of the roles of this team.
+
+You will need to authenticate with the API, for information about authentication read the [API authentication](/konnect/api/index/#authentication) instructions.
```

Comment:
```suggestion
You must authenticate with the API, for information about authentication read the [API authentication](/konnect/api/index/#authentication) instructions.
```
I feel like the "why" is missing here, ex. why do you need to authenticate with the API? Is this just so custom teams work?

Reply:
Or this feels like a prereq, so maybe make a Prereq section with this in it?